datacats/docker.py

```diff
@@ -181,7 +182,8 @@ def web_command(command, ro=None, rw=None, links=None, stream_output=None,
         command=command,
         volumes=binds_to_volumes(binds),
         detach=False,
-        host_config=create_host_config(binds=binds))
+        host_config=create_host_config(binds=binds),
+        entrypoint=entrypoint)
```

Comment:
So docker entrypoint is the script that runs when an image is used to create a container, right?